Apps/Src/Temperature.c

```diff
@@ -28,9 +31,17 @@ static int32_t maxTemperature;
 // 0 if discharging 1 if charging
 static uint8_t ChargingState;
 
+// simulator bypasses ltc driver
+#ifndef SIMULATION
 // Interface to communicate with LTC6811 (Register values)
 // Temperature.c uses auxiliary registers to view ADC data and COM register for I2C with LTC1380 MUX
 static cell_asic *Minions;
```

Comment:
So would Minions not be used if the simulator was running? Could the simulator not write to certain locations within the Minions struct or is that because the implementation would get too complicated? I'm wondering since some applications return data straight from the struct.

Reply:
Supporting arbitrary reads from Minions would require building a model of the LTC6811 to make sure all of the registers in the Minions struct are correct